### ISSUE TITLE
Annoying matplotlib deprecation warnings

### ISSUE THREAD

**Issue by gridley** [mit-crpg/OpenMOC#441]
To see the issue, run any case which makes plots using matplotlib 3 or above. My particular version is 3.1.1.

How can we make this go away?

```
/home/gavin/.local/lib/python3.6/site-packages/openmoc/plotter.py:1323: MatplotlibDeprecationWarning: 
The examples.directory rcparam was deprecated in Matplotlib 3.0 and will be removed in 3.2. In the future, examples will be found relative to the 'datapath' directory.
  curr_rc = dict(matplotlib.rcParams)
/usr/lib/python3.6/_collections_abc.py:841: MatplotlibDeprecationWarning: 
The examples.directory rcparam was deprecated in Matplotlib 3.0 and will be removed in 3.2. In the future, examples will be found relative to the 'datapath' directory.
  self[key] = other[key]
/usr/lib/python3.6/_collections_abc.py:841: MatplotlibDeprecationWarning: 
The savefig.frameon rcparam was deprecated in Matplotlib 3.1 and will be removed in 3.3.
  self[key] = other[key]
/usr/lib/python3.6/_collections_abc.py:841: MatplotlibDeprecationWarning: 
The text.latex.unicode rcparam was deprecated in Matplotlib 3.0 and will be removed in 3.2.
  self[key] = other[key]
/usr/lib/python3.6/_collections_abc.py:841: MatplotlibDeprecationWarning: 
The verbose.fileo rcparam was deprecated in Matplotlib 3.1 and will be removed in 3.3.
  self[key] = other[key]
/usr/lib/python3.6/_collections_abc.py:841: MatplotlibDeprecationWarning: 
The verbose.level rcparam was deprecated in Matplotlib 3.1 and will be removed in 3.3.

```

Seems related to [this](https://github.com/pyviz/holoviews/issues/2963).

**Comment by GiudGiud** [comment]
The issue you linked to has a fix. Making a PR soon

**Comment by gridley** [comment]
Nice! I was thinking about doing this so thanks for the heads up.

**Comment by GiudGiud** [comment]
Addressed in #444
